setup.py

```diff
@@ -0,0 +1,114 @@
+"""
+Neuraxle
+====================================
+This is the core of Neuraxle. Most pipeline steps derive (inherit) from those classes. They are worth noticing.
+..
+    Copyright 2019, Neuraxio Inc.
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+        http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing, software
```

Comment:
I see that you often collapse the newlines in those legal comments. I'd prefer them to be uniform with all the spacing as in the following original file: 
https://github.com/Neuraxio/Neuraxle/blob/master/setup.py

Also don't forget to add this in other files where it might be missing in this new project.